programs/conditional_vault/src/lib.rs

```diff
@@ -265,17 +275,22 @@ pub mod conditional_vault {
         Ok(())
     }
 
-    pub fn mint_conditional_tokens(ctx: Context<MintConditionalTokens>, amount: u64) -> Result<()> {
+    pub fn mint_conditional_tokens(ctx: Context<InteractWithVault>, amount: u64) -> Result<()> {
```

Comment:
It probably doesn't matter and it wasn't in the original account struct but I wonder if there's any issues or downside with limiting minting to when the market is active?

Reply:
I think you want people to be able to capture arbs in expired markets when they exist

Reply:
Fair enough